pages/docs/contributing/adding-a-new-app.mdx

```diff
@@ -38,122 +44,99 @@ Create a new branch for your app.
 git checkout -b app/<app-name>
 ```
 
-## Create the app files
+### Create the app files
 
-Each app requires at least the following files:
+Each app requires at least the files:
 
 - A `docker-compose.yml` file to run your app
 - A `config.json` file to configure your app
 - A description in markdown format
-- A logo in jpg format (512 x 512px)
+- A logo in jpg format (should be atleast `200 x 200` pixels)
```

Comment:
why 200?

Reply:
With glance which added the image was 200x200 and it looked good on the appstore. I think it should be the minimum.